src/vdaf/prio3_instances.rs

```diff
@@ -0,0 +1,224 @@
+//! Implementation of the Prio3 instances from [[draft-irtf-cfrg-vdaf-01]].
+//! These use the internal PRG.
+//!
+//! - [`Prio3Aes128Count`] for aggregating a counter (*)
+//! - [`Prio3Aes128CountVec`] for aggregating a vector of counters
+//! - [`Prio3Aes128Sum`] for copmputing the sum of integers (*)
+//! - [`Prio3Aes128Histogram`] for estimating a distribution via a histogram (*)
+//!
+use crate::field::{Field128, Field64};
+#[cfg(feature = "multithreaded")]
+use crate::flp::gadgets::ParallelSumMultithreaded;
+use crate::flp::gadgets::{BlindPolyEval, ParallelSum};
+use crate::flp::types::{Count, CountVec, Histogram, Sum};
+use crate::vdaf::prg::PrgAes128;
+use crate::vdaf::prio3::check_num_aggregators;
+use crate::vdaf::prio3::Prio3;
+use crate::vdaf::VdafError;
+
+/// The count type. Each measurement is an integer in `[0,2)` and the aggregate result is the sum.
+pub type Prio3Aes128Count = Prio3<Count<Field64>, PrgAes128, 16>;
+
+impl Prio3Aes128Count {
+    /// Construct an instance of Prio3Aes128Count with the given number of aggregators.
+    pub fn new_aes128_count(num_aggregators: u8) -> Result<Self, VdafError> {
+        check_num_aggregators(num_aggregators)?;
+
+        Prio3::new(num_aggregators, Count::new())
```

Comment:
I like that these constructors now all call the common constructor! Since `Prio3::new` calls `check_num_aggregators`, can we remove that call from here and the other constructors for the `Prio3` type aliases?